server/routes/induction/create/checkYourAnswersCreateController.test.ts

```diff
@@ -1,32 +1,44 @@
+import createError from 'http-errors'
 import { NextFunction, Request, Response } from 'express'
 import type { SessionData } from 'express-session'
 import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
 import { aShortQuestionSetInductionDto } from '../../../testsupport/inductionDtoTestDataBuilder'
 import CheckYourAnswersCreateController from './checkYourAnswersCreateController'
+import { aShortQuestionSetCreateInductionDto } from '../../../testsupport/createInductionDtoTestDataBuilder'
+import InductionService from '../../../services/inductionService'
+import toCreateOrUpdateInductionDto from '../../../data/mappers/createOrUpdateInductionDtoMapper'
+
+jest.mock('../../../services/inductionService')
+jest.mock('../../../data/mappers/createOrUpdateInductionDtoMapper')
 
 describe('checkYourAnswersCreateController', () => {
-  const controller = new CheckYourAnswersCreateController()
+  const mockedCreateOrUpdateInductionDtoMapper = toCreateOrUpdateInductionDto as jest.MockedFunction<
+    typeof toCreateOrUpdateInductionDto
+  >
+
+  const inductionService = new InductionService(null) as jest.Mocked<InductionService>
+  const controller = new CheckYourAnswersCreateController(inductionService)
 
   const prisonNumber = 'A1234BC'
   const prisonerSummary = aValidPrisonerSummary()
 
-  const req = {
-    session: {} as SessionData,
-    user: {} as Express.User,
-    params: {} as Record<string, string>,
-    flash: jest.fn(),
-  }
-  const res = {
-    redirect: jest.fn(),
-    render: jest.fn(),
-  }
+  let req: Request
+  let res: Response
+
   const next = jest.fn()
 
   beforeEach(() => {
     jest.resetAllMocks()
-    req.session = { prisonerSummary } as SessionData
-    req.user = { token: 'some-token' } as Express.User
-    req.params = { prisonNumber }
+    req = {
```

Comment:
I've followed this more concise approach that I saw you used in one of your PRs @leejacobson-moj  re: setting up the mock request and response 👍